app/controllers/solidus_bolt/accounts_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'base64'
```

Comment:
I don't think we need it here, because it's required from `SolidusBolt::BaseController`

Reply:
removed in https://github.com/nebulab/solidus_bolt/commit/cbad318944309509197f3931924794104865de47